elements/classpower.lua

```diff
@@ -113,10 +113,17 @@ local function Update(self, event, unit, powerType)
 			mod = UnitPowerDisplayMod(ClassPowerID)
 		end
 
+		local numActive = math.ceil(mod == 0 and 0 or cur / mod)
```

Comment:
You could spare the call to math.ceil by `local numActive = (mod == 0 and 0 or cur / mod) + 0.9`